composer.json

```diff
@@ -9,12 +9,19 @@
         "illuminate/support": "^8.0"
     },
     "require-dev": {
+        "ext-dom": "*",
```

Comment:
I guess this is no longer needed?

Reply:
Correct - just overlooked that after I looked at how your action worked to match that. 😄

Reply:
I'll make sure to address removing this when I remove the `"repositories"` key after the PR is merged in at blade-icons.